src/components/v5/common/Fields/Switch/Switch.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { FC } from 'react';
+
+import clsx from 'clsx';
+import styles from './Switch.module.css';
+import { SwitchProps } from './types';
+
+const displayName = 'v5.common.Fields.Switch';
+
+const Switch: FC<SwitchProps> = ({ id = 'switch-id', isDisabled }) => (
+  <label
+    htmlFor={id}
+    className={clsx('relative', {
+      'cursor-not-allowed': isDisabled,
```

Comment:
As I can see we used 'pointer-events-none' so far for the disabled state. Please, correct it to be consistent